Framework/include/QualityControl/InfrastructureGenerator.h

```diff
@@ -178,6 +178,12 @@ class InfrastructureGenerator
 
   static void printVersion();
 
+  /**
+   * If an aggregator has the same name as a check, we throw a runtime_error.
+   * @param infrastructureSpec
+   */
+  static void throwIfNameClash(const InfrastructureSpec& infrastructureSpec);
```

Comment:
The name and placement suggests that it checks for any name clashes. Could you please make it more specific or document so that it indeed is supposed to check any name clashes?

Reply:
yes, indeed. Done.